feg/gateway/services/s8_proxy/servicers/senders.go

```diff
@@ -93,3 +93,18 @@ func (s *S8Proxy) sendAndReceiveCreateBearerResponse(res *protos.CreateBearerRes
 	}
 	return &orc8r_protos.Void{}, err
 }
+
+// sendAndReceiveDeleteBearerResponse
+func (s *S8Proxy) sendAndReceiveDeleteBearerResponse(res *protos.DeleteBearerResponsePgw,
+	cPgwUDPAddr *net.UDPAddr,
+	cbResMsg message.Message) (*orc8r_protos.Void, error) {
+	glog.V(2).Infof("Send Delete Bearer Response (grpc) to %s:\n%s", cPgwUDPAddr,
+		res)
+	glog.V(2).Infof("Send Delete Bearer Response (gtp) to %s:\n%s",
```

Comment:
Why do we need both logs?

Reply:
this is because go-gtp doesn't show all the content. Since those messages are new I prefer to have the plain value (grpc) and the encoded value (gtp) visible for easy debugging. 

Note this is only visible on level 2